src/validators.nix

```diff
@@ -77,6 +77,7 @@ in {
         flake = false;
```

Comment:
Remove.

Reply:
I was thinking we would begin the deprecation process in a separate PR, but did you want to target that for this one?

Reply:
Yeah, we need to deprecate the old interface already, so let's not do multiple deprecations that all overlap in short sequence, see: https://github.com/divnix/std/pull/206#issuecomment-1358683235

Reply:
implemented in 26c51e2e90b9b93d03582aca135409295a27814e